### ISSUE TITLE
Android not working

### ISSUE THREAD

**Issue by yongloon** [anakornk/web3WithReactNative#1]
![screen shot 2019-01-10 at 1 40 58 am](https://user-images.githubusercontent.com/12860854/50917573-d0030280-1478-11e9-83ad-41040185d5f1.png)
Hi.. i face this error when run on android device.

**Comment by anakornk** [comment]
@nyl9488 The problem was with when we import “whatwg-url” in the globals.js file. 
```
import { URL, URLSearchParams } from "whatwg-url";
global.URL = URL;
global.URLSearchParams = URLSearchParams;
```
We needed this to fix the issue “Error: Creating URL objects is not supported yet. ” @https://github.com/facebook/react-native/issues/16434

Thank you for pointing it out, did not know that it wasn’t working on android.
We need an alternative that works for android. Replace the above with this and it should work.
```
import url from "url";
global.URL = class URL {
  constructor(inputUrl) {
    return url.parse(inputUrl);
  }
};
```
I have created a bug-fix branch #2 . Please let me know how it goes.

**Comment by yongloon** [comment]
thanks... working now. :)
